src/components/pages/login-page/components/logo-register-block/LogoRegisterBlock.tsx

```diff
@@ -1,36 +1,39 @@
+import { Box, Typography } from '@mui/material';
 import Image from 'next/image';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
 
 import Button from '@/components/common/ui/button';
 
-import styles from './LeftBlock.module.scss';
+import * as styles from './LogoRegisterBlock.styles';
 
-const LeftBlock = () => {
+const LogoRegisterBlock = () => {
   const { push } = useRouter();
 
   return (
-    <div className={styles['left-block']}>
+    <Box sx={styles.logoRegisterBlock}>
       <Link href="/">
         <Image
-          className={styles['login-logo']}
+          style={{ margin: 0, padding: '8px' }}
           src="/images/login-page/new-logo.png"
           alt="fict advisor logo"
           priority
           width={300}
           height={54}
         />
       </Link>
-      <h3 className={styles['register-text']}>Ти ще не з нами? Приєднуйся!</h3>
+      <Typography variant="h3" sx={styles.registerText}>
```

Comment:
as your font doesn't changes with width size, remove styles for fonts in `registerText` and set correct `variant` for `Typography`